### PR TITLE
Milestone: mobile menu 👌

### PR DESCRIPTION
**Pull request by ahmedeid6842**
## Added a menu to the mobile version :eyes:
- Apply javascript query selectors to control the component 
- Use JavaScript events listener `click`.
- Each item on the menu point to its relative section
- Add CSS property `visibility` to control the menu's visibility 
- Updated a README documentation

PeacefulNecessaryGrasshopper-max-1mb

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -38,10 +37,41 @@ header {
...
-.nav-list {
+.mobile-menu {
   display: none;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  position: fixed;
+  visibility: hidden;
+}
```
Comment (TadesseDev) [inline styles/main.css:51]: - [ ] Your design is almost exceptional team 👏. Here just to make sure we are following the design guideline 100%, Please use the appropriate color for your navbar/hamburger icon. 

### Your design 
image
### Figma design 
image

File: index.html [hunk 2]
```diff
@@ -15,233 +15,243 @@
...
-        <nav>
+        <div class="nav-menu">
+            <img src="./images/menu.svg" alt="menu-icon" class="menu nav-menu">
+        </div>
 
+        <nav class="mobile-menu">
+            <div class="close-menu">
+                <img src="./images/icons8-close.svg" alt="menu-close" class="close-menu-icon">
+            </div>
             <ul class="nav-list">
```
Comment (TadesseDev) [inline index.html:26]: - [ ] Please consider adding the horizontal line which is depicted in the given design, You are not required to implement the functionality at this point but to follow the design guideline we need to add the line. 

### Your design 
image


### Figma design 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitHub Flow  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## STATUS: APPROVED 👍🎯  
Hello @ahmedeid6842 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ requested changes implemented
✔️ linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmedeid6842** [comment]
# 🎉🎉🥳🥳

7fb4d55fa09fc4f5426e6d5934d1d02f

**Pull request merged**